docs/docs/GettingStarted.md

```diff
@@ -0,0 +1,152 @@
+---
+category: Documentation
+categoryindex: 1
+index: 1
+---
+# Getting Started
+
+## Quick install
+
+* Command line: `dotnet tool install -g fantomas`
```

Comment:
I'd split this up into multiple pages.
The first page can capture everything about the command-line tool.
Then you have a page for each editor and one for FAKE.
It isn't bad that some of these pages are rather short.